common.flag.mk

```diff
@@ -4,38 +4,28 @@ DYNLINK ?= false
 # For COMMON
 #=================================================
 
-COMMON_FLAGS     = -fPIC -Werror -Wall
+COMMON_FLAGS     = -fPIC -g #-Werror -Wall
```

Comment:
I would add `-Wall -Wextra` and only remove the `-Werror`. Then, and this is the most important part, triage and fix the warnings. Definitely let me know if you have any questions about which warnings are important.